spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/ThresholdedReLU.scala

```diff
@@ -39,7 +39,8 @@ import scala.reflect.ClassTag
 class ThresholdedReLU[T: ClassTag](
    val theta: Double = 1.0,
    val inputShape: Shape = null)(implicit ev: TensorNumeric[T])
-  extends KerasLayer[Tensor[T], Tensor[T], T](KerasLayer.addBatch(inputShape)) {
+  extends KerasLayer[Tensor[T], Tensor[T], T](KerasLayer.addBatch(inputShape))
+    with IdentityOutputShape{
```

Comment:
Add a space between IdentityOutputShape and {
So that every new layer is formatted.